src/Http/Controllers/CP/Forms/SubmissionActionController.php

```diff
@@ -14,4 +17,17 @@ protected function getSelectedItems($items, $context)
             return $form->submission($item);
         });
     }
+
+    protected function getItemData($submission, $context): array
```

Comment:
Just thinking about this now...

Since we're not showing actions on the publish form page, maybe this should just return an empty array for the time being and we can implement this if/when we get around to it.